run--01a--prep-geo-data.Rmd

```diff
@@ -584,9 +799,9 @@ if (exists("my_aux_geo")) {
 
 ```{r collect and save GIS data}
 geo_objects <- c("my_state_fip", "geo_crosswalk", "geo_crosswalk_zcta", "geo_crosswalk_sd", "tractShp", "pumaShp", "countyShp", "zctaShp", "schoolShp", "leaflet_base_map")
-if (exists("auxShp")) geo_objects <- c(geo_objects, "auxShp", "geo_crosswalk_aux")
+if (exists("auxShp"))        geo_objects <- c(geo_objects, "auxShp", "geo_crosswalk_aux")
 if (exists("my_county_fip")) geo_objects <- c(geo_objects, "my_county_fip")
-
+if (my_state_abbr == "IL")   geo_objects <- c(geo_objects, "geo_crosswalk_b5")
```

Comment:
As above -- this shouldn't be in `main`. We should take time to expand the `aux` specification.